Description: Make upstream cgconfig initscript cope if .deb is removed but not purged
Author: Nick Leverton <nick@leverton.org>
Bug-Debian: TBA

Index: libcgroup-0.38/scripts/init.d/cgconfig.in
===================================================================
--- libcgroup-0.38.orig/scripts/init.d/cgconfig.in	2011-12-06 09:42:15.000000000 +0000
+++ libcgroup-0.38/scripts/init.d/cgconfig.in	2013-04-07 19:46:09.000000000 +0100
@@ -36,6 +36,9 @@
 servicename=cgconfig
 lockfile=/var/lock/subsys/$servicename
 
+# Sanity check
+[ "`which cgcreate`" -a "`which cgclassify`" ] || exit 1
+
 #
 # Source LSB routines
 #
